src/native/eventpipe/ep-session-provider.h

```diff
@@ -11,6 +11,90 @@
 #endif
 #include "ep-getter-setter.h"
 
+/*
+ * EventPipeSessionProviderTracepoint.
+ */
+
+#if defined(EP_INLINE_GETTER_SETTER) || defined(EP_IMPL_EP_GETTER_SETTER)
+struct _EventPipeSessionProviderTracepoint {
+#else
+struct _EventPipeSessionProviderTracepoint_Internal {
+#endif
+	const ep_char8_t *tracepoint_format;
```

Comment:
Guess this shouldn't be const since the type owns this field and will free it in a function implemnted in ep-session-provider.c meaning it has direct access to the member.